src/sql/base/browser/ui/buttonMenu/buttonMenu.css

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.monaco-dropdown {
+	height: auto;
```

Comment:
These are going to affect ALL dropdowns that use this class - which are most. Either scope this down or put it in the CSS for the actual dropdown component

Reply:
First item used to add cell as code or markdown:
![image](https://user-images.githubusercontent.com/12279161/81205614-ceb7ec00-8f7f-11ea-8c9b-ae560c902ffb.png)

Add cell menu open:
![image](https://user-images.githubusercontent.com/12279161/81205552-b778fe80-8f7f-11ea-9b5b-b9c47ccb1c81.png)

The end goal here will be use styled elements to replace the OS rendered open menu for this item and all other Notebook UI that opens a menu.

As for functionality:

- Label text with (optionally) an icon preceding the text and a dropdown arrow following the text. No select box border.
- Dropdown items consisting of actions preceded (optionally) by an icon each (see comp below)
- Reusability across Notebook UI including upcoming cell toolbar.

Here's a snap fo the comp:
![image](https://user-images.githubusercontent.com/12279161/81206360-d926b580-8f80-11ea-8f3f-f5601f1275a7.png)

Reply:
I'll scope it down using a new class.

Reply:
@halerankin I would still prefer this to be scoped down to just this component. in-preview is too generic and could possibly be something applied to another component which then would affect all dropdowns below it.

Something like this is preferable (change as needed for given DOM structure)

`.button-menu .in-preview .monaco-dropdown`

Reply:
Scoped.